urllib3/util/connection.py

```diff
@@ -1,16 +1,19 @@
 from __future__ import absolute_import
 import socket
+from urllib3.util.rfc6555 import (
+    happy_eyeballs_algorithm
+)
 from .wait import wait_for_read
 from .selectors import HAS_SELECT, SelectorError
 
+ENABLE_HAPPY_EYEBALLS = True
```

Comment:
If they need to disable happy eyeballs we have failed miserably. Regardless, we should make it possible. Let's make it private (`_ENABLE_HAPPY_EYEBALLS`) and put a comment above it indicating why it's there.